.travis.yml

```diff
@@ -4,7 +4,7 @@ python:
   - "3.6"
 # command to install dependencies
 install:
-  - pip install -e .
+  - make dependancies
```

Comment:
dependencies

Reply:
Fixed in https://github.com/jdow/ldap-access-log-humanizer/pull/28